src/client/src/ui/spotTile/reducer.ts

```diff
@@ -25,7 +25,7 @@ export const spotTileDataReducer = (state: any = {}, action) => {
       return updateSpotTile(state, payload.CurrencyPair, { isTradeExecutionInFlight: true })
 
     case ACTION_TYPES.TRADE_EXECUTED:
-      const symbol = payload.trade.symbol
+      const symbol = payload.trade.symbol || payload.trade.CurrencyPair
```

Comment:
seems a bit strange that an action can have two types of payload.

Reply:
yes, very. but this will require a more detailed investigation